backend/news-service/src/Requests.js

```diff
@@ -1,110 +1,97 @@
 const R = require('request');
 const db = require('./controllers/DatabaseInterface');
+const SH = require('./ScraperHandler');
 require('dotenv').config();
 
-const { MIDDLEWARE_HOST, MIDDLEWARE_PORT, SCRAPER_PORT } = process.env;
+const {
+  MIDDLEWARE_HOST, MIDDLEWARE_PORT, SVT_PORT, POLISEN_PORT,
+} = process.env;
 const MIDDLEWARE_ROUTE = '/complete_request';
 const COMPLETE_REQUEST_URL = `http://${MIDDLEWARE_HOST}:${MIDDLEWARE_PORT}${MIDDLEWARE_ROUTE}`;
-const SCRAPER_URL = `http://localhost:${SCRAPER_PORT}/getnews/daterange`;
+const SCRAPER_URL_SVT = `http://localhost:${SVT_PORT}/getnews/daterange`;
```

Comment:
Will need a host parameter as well.

Reply:
Will be solved later when kubernetes is used since things will need to change then anyway